Shevchuk_Maksym/README.md

```diff
@@ -0,0 +1,93 @@
+Discord Registration and Usage Guide
```

Comment:
Please consider formatting the document properly. Use different styles to mark the headings, numbered lists for the procedures to make the list of steps easy to scan, and so on. Also, visual aids would be of great value.
Refer to https://learn.microsoft.com/en-us/style-guide/procedures-instructions/ for more info.